src/providers/python.rs

```diff
@@ -323,6 +322,7 @@ impl PythonProvider {
         ))
     }
 
+    #[allow(dead_code)]
```

Comment:
Do we need this? Is there a time when this function is never called?

Reply:
It currently is never called, but I didn't want to remove it since it may be useful in the future.